### DISCUSSION TITLE
Support referencing spa loading template from base layer

### DISCUSSION THREAD

**Discussion by trc-mathieu** [nuxt/nuxt#24274] [Ideas]
I have a use case where we are reusing the same base layer for all the apps we build in our organization. I would like to reuse the loading indicator that we normally put in the `app/spa-loading-template.html` or at least be able to reference it in the nuxt.config.
As reference, we only look at the `srcDir`:
https://github.com/nuxt/nuxt/blob/8bff8136c16a6e42e7f34590b750b2b6a60730bc/packages/nuxt/src/core/nitro.ts#L517-L520

If it could automatically take the `app/spa-loading-template.html` from the layer, that would be awesome, or allow us to configure it in the nuxt.config.

**Comment by TheAlexLichter** [comment]
Opened an [issue here](https://github.com/nuxt/nuxt/issues/24275) instead 👍🏻
